.github/workflows/deploy.yml

```diff
@@ -6,8 +6,12 @@ jobs:
     name: Deploy
     runs-on: ubuntu-latest
     steps:
-      - name: Dump event
-        run: jq . $GITHUB_EVENT_PATH
+      - name: Echo the client name
+        run: echo {{github.event.client_payload.repo}}
+        if: github.event.action == 'build_application'
```

Comment:
Can we also keep the original step? I realized after filing #24 that it will probably be useful to look at the full event in case something unexpected goes wrong.

Reply:
Sure. I'm updating it now but it won't be committed until I've run a couple more tests on the live instance

Reply:
Fixed in b870842